### ISSUE TITLE
How much would it cost to keep this running? 

### ISSUE THREAD

**Issue by deadbydonovan** [C-Nedelcu/talk-to-chatgpt#249]
Why can't this be a service that we pay for? I'd pay $5 a month to keep using this extension.

**Comment by hoshizorista** [comment]
Not necessary, there are multiple people that use this extension and we will struggle to keep it alive, I have a fork of the extension compatible with GPT4o, check it out if you find difficult to apply the changes to make it work that are discussed on the previous issues!

**Comment by i3dman420311** [comment]
@hoshizorista can this be made to auto start? i'd like to not have to press the start button anymore.

**Comment by hoshizorista** [comment]
@i3dman420311 on content.js look for 	

```
// Start key detection
	jQuery(document).on('keydown', function (e) {
```

and above it, paste 

```
	jQuery(document).ready(function() {
		console.log('Document is ready, starting Talk-To-ChatGPT');
		CN_StartTTGPT();
	});
```

it will auto-start once chatgpt.com its opened

**Comment by i3dman420311** [comment]
@hoshizorista This is fantastic! Thank you!

**Comment by i3dman420311** [comment]
@hoshizorista Is it possible to keep the microphone always active.
I find that the software will stop listening for unknown reasons other than the tab is not the active tab or chrome is minimized.

**Comment by i3dman420311** [comment]
I've also noticed voice playback does not work , unless i go in to settings first then back out.

**Comment by hoshizorista** [comment]
@i3dman420311 i tested the autostart function and it works fine, it keeps listening until you speak and playback is autoenabled, perhaps you are having an isolated issue, add me to discord if you need more help, my tag is exile9904

**Comment by i3dman420311** [comment]
@hoshizorista friend request sent in discord. I also have video of my issues.

**Comment by deadbydonovan** [comment]
@hoshizorista Thanks for the fix last month, are you having any new issues? The extension no longer autosends or plays the A.I. voice back for responses. I think the introduction of 4o mini messed this up.

**Comment by hoshizorista** [comment]
@deadbydonovan gonna check it out, havent seen it in a while, however feel free to try the fork i made, its way more updated and fixed, try with that one and lmk if issue persists

**Comment by deadbydonovan** [comment]
@hoshizorista Thank you for the quick response, I tried it, same issue, I think I was already using your fork from a month ago. I got this error message.        

 console.error("[CN_SendMessage] Send button not found."); 
 
 But also, manually clicking send, the response back doesn't play the audio.

**Comment by deadbydonovan** [comment]
@hoshizorista Just curious, is the extension working for you or anyone else since the 4o mini update?

**Comment by hoshizorista** [comment]
@deadbydonovan Nope, it was dead, openai loves to change selectors on every model update, I just updated it and I briefly tested it, seems to be fine! I made a fork of this extension which I keep updated to work with latest updates, check it out bro (there you can download the latest zip with the extension ready to install)

https://github.com/hoshizorista/talk-to-chatgpt-fixed-gpt4o/releases
